ontology/ro.md

```diff
@@ -8,7 +8,6 @@ build:
   method: vcs
   infallible: 1
 canonical: ro.owl
-aberowl_id: relo
```

Comment:
http://aber-owl.net/ontology/RO seems to work properly

Reply:
Absolutely, thank you for checking!